waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -342,7 +342,8 @@ genViteConfig spec = return $ C.mkTmplFdWithData tmplFile tmplData
     tmplData =
       object
         [ "customViteConfig" .= jsImportToImportJson (makeCustomViteConfigJsImport <$> AS.customViteConfigPath spec),
-          "baseDir" .= WebApp.getBaseDir spec
+          "baseDir" .= WebApp.getBaseDir spec,
```

Comment:
I commented on this before, but same as for baseDir, I don't think deafultClientPort should be in `WebApp`.